projects/scripts/adi_project_xilinx.tcl

```diff
@@ -324,10 +338,10 @@ proc adi_project_run {project_name} {
   launch_runs impl_1 -to_step write_bitstream
   wait_on_run impl_1
   open_run impl_1
-  report_timing_summary -warn_on_violation -file timing_impl.log
+  report_timing_summary -warn_on_violation -file ${actual_project_name}_timing_impl.log
 
   if {[info exists ::env(ADI_GENERATE_UTILIZATION)]} {
-    set csv_file resource_utilization.csv
+    set csv_file ${actual_project_name}_resource_utilization.csv
     if {[ catch {
```

Comment:
There is a log file generated for each IP from 'IP_list', but those will be generated directly in project folder.
They need to be generated also in sub-folder created from parameters' values.
See "set output_file ${IP_name}_resource_utilization.log" line.